### ISSUE TITLE
General Retrospective for April 2023 Releases

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/adoptium#225]
**Summary**

A retrospective for all efforts surrounding the titular releases. 

All community members are welcome to contribute to the agenda via comments below.

This will be a virtual meeting after the release, with at least a week of notice in the #release Slack channel.

On the day of the meeting the agenda items will be iterated over, with a list of actions added in a comment at/near the end.

Invited: Everyone.

**Time, Date, and URL**

Time: 
Date: 

URL: 
Dial-in: 
PIN: 

**Details**

Retrospective Owner Tasks (in order):

- [ ] Post this issue's URL on the \#Release Slack channel around the start of the new release.
- [ ] Copy actions from the previous retrospective into this issue, while ignoring actions that are ticked or have issue links (i.e. clearly-completed actions).
- [ ] Wait until 90% of all builds have been released, with no clear signs of additional respins.
- [ ] Announce the retrospective Slack call's date + time on \#Release at least one full week in advance, and send out meeting invites.
- [ ] Host the slack call for the retrospective, including:
  - [ ] Iterating over the actions from the previous retrospective issue, ticking off completed items.
  - [ ] Iterate over the agenda, ensuring everything gets debated.
  - [ ] Create a clear list of actions at the end of the retrospective, including volunteer names.
- [ ] Create a new retrospective issue for the next release.
- [ ] Set yourself a calendar reminder so that you remember to commence step 1 (in the new issue) just before the next release.
- [ ] Close this issue.

**Comment by smlambert** [comment]
JDK20, we still launched the builds for the 3 platforms we knew were not in plan for the release (AIX, s390x, arm32) and I caught myself triaging one, before I remembered that I did not need to put effort there.

**Comment by smlambert** [comment]
Reorganize the [template](https://github.com/adoptium/adoptium/blob/main/.github/ISSUE_TEMPLATE/release-status.md) for the release status issue to put the 2 additional primary platforms (aarch64 Linux, aarch64 Mac) at the top of the table and in bold.

**Comment by smlambert** [comment]
Extended.openjdk on AIX is being run in parallel mode across 3 nodes, but there is only a single node online with AIX 72 label, so it should not be launched in parallel and hits 15 hr time_limit without completion.

**Comment by smlambert** [comment]
[2023Q1_Adoptium Program plan_By Quarter_Community Call.pdf](https://github.com/adoptium/adoptium/files/11334933/2023Q1_Adoptium.Program.plan_By.Quarter_Community.Call.pdf)

**Comment by smlambert** [comment]
[2023Q1_Adoptium Program plan_By Quarter_Community Call.pdf](https://github.com/adoptium/adoptium/files/11334970/2023Q1_Adoptium.Program.plan_By.Quarter_Community.Call.pdf)

**Comment by steelhead31** [comment]
Improve/add installers specific checklist/matrix element, to ensure all combinations of version(8,11,17,etc) , type(JDK/JRE) , architectures & distros are covered.

**Comment by andrew-m-leonard** [comment]
Forgot to specify overridePublishName parameter, for jdk8 arm32 build, so job artifacts named wrong:
 OpenJDK8U-debugimage_arm_linux_hotspot_8u372b07-aarch32-20230426.tar.gz

**Comment by andrew-m-leonard** [comment]
Improve installers & dockerhub release "checklist", maybe add issue template to each repo, and link for main checklist..

**Comment by steelhead31** [comment]
Issue for checklists : #227

**Comment by adamfarley** [comment]
Actions have been copied into the [July General Retrospective](https://github.com/adoptium/adoptium/issues/240#issuecomment-1662060816). Closing this issue.

**Comment by adamfarley** [comment]
@andrew-m-leonard - Please close, thx.
